doc/user/layouts/partials/create-source/connector/file/details.html

```diff
@@ -5,3 +5,12 @@
 ```sql
 CREATE SOURCE server_source FROM FILE '/Users/sean/server.log'...
 ```
+
+#### Compression
```

Comment:
Because the items under this heading are distinct, mostly-atomic units of information, I'd format these as an unordered list.